src/core/Effect.cpp

```diff
@@ -52,9 +52,11 @@ Effect::Effect( const Plugin::Descriptor * _desc,
 	m_autoQuitModel( 1.0f, 1.0f, 8000.0f, 100.0f, 1.0f, this, tr( "Decay" ) ),
 	m_autoQuitDisabled( false )
 {
+	m_wetDryModel.setCenterValue( 0 );
```

Comment:
Also, you can remove the spaces within the brackets.

```suggestion
	m_wetDryModel.setCenterValue(0);
```

Reply:
Of the five other uses of setCenterValue, four use whitespaces.
![image](https://github.com/LMMS/lmms/assets/76674645/d4359231-32b2-48e2-98a9-193dde608c68)
![image](https://github.com/LMMS/lmms/assets/76674645/4744d373-7fd8-4e58-ab83-c67a435faa84)

Having just noticed the [convention](https://github.com/LMMS/lmms/wiki/Coding-conventions#whitespace) on spaces in parentheses, I'll fix

Reply:
Only touch what the PR needs to touch. Those other matches aren't part of the PR, so no need to change them. 👍